src/features/offer/OfferContainer.tsx

```diff
@@ -64,15 +66,28 @@ const OfferContainer = () => {
   const harbor = getHarbor(data);
   const boat = getBoat(data);
 
-  const handleClickSelect = (berthId: string) => {
+  const handleClickSelect = (berth: BerthData) => {
     createBerthLease({
       variables: {
         input: {
           applicationId: applicationId || '',
-          berthId,
+          berthId: berth.id,
         },
       },
-    }).then(() => history.push('/applications'));
+    }).then(() => {
+      history.push('/applications');
+      hdsToast({
+        type: 'success',
+        autoDismiss: false,
```

Comment:
Should we autoDismiss it?

Reply:
The autodismiss is so quick I didn't have time to verify from the text that it was actually assigned correctly.

Reply:
Fair enough.

Reply:
Added a way to configure the dismiss time and increased it a bit for this toast.